.github/workflows/merge.yaml

```diff
@@ -106,7 +106,7 @@ jobs:
             gcloud artifacts docker tags add ${docker_registry}/${image}:${docker_pr_tag} ${docker_registry}/${image}:${docker_release_latest_tag}
             if [ "${base_branch}" == "master" ]; then
               echo "Tagging ${image}:latest"
-              gcloud artifacts docker tags add ${docker_registry}/${image}:${docker_pr_tag} ${docker_registry}/${image}:latest
+              # gcloud artifacts docker tags add ${docker_registry}/${image}:${docker_pr_tag} ${docker_registry}/${image}:latest
```

Comment:
@ausias-armesto This is not correct, right?

Reply:
Needs to be uncommented

Reply:
Fixed in eee7f991fdf43b0a1ed2f40b545f88b66dade679